docs2/site/docs/getting-started/introduction.md

```diff
@@ -217,12 +217,12 @@ public class Query
 [GraphQLMetadata("Droid", IsTypeOf=typeof(Droid))]
 public class DroidType
 {
-  public string Id(Droid droid) => droid.Id;
-  public string Name(Droid droid) => droid.Name;
+  public string Id([FromSource] Droid droid) => droid.Id;
```

Comment:
What will happen if you do not specify this attribute?

Reply:
It would be treated as an input argument, e.g. `GetArgument<Droid>(“droid”)`